src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.ps1

```diff
@@ -196,4 +196,266 @@ $query1Scope = @(
     Assert-NotNull $sucGet.UpdateConfiguration.Targets.AzureQueries "Update targets  azureQueries list  null"
     Assert-AreEqual $sucGet.UpdateConfiguration.Targets.AzureQueries.Count  1 "Update targets  doesn't have the correct number of azure queries"
    
+ }
+
+ <#
+.SYNOPSIS
+Tests create new automation variable with string value.
+#>
+function Test-CreateAndGetSoftwareUpdateConfigurationWithAzureDynamicGroupsOnly
+{
+    $name = "DG-suc-04"
+    $startTime = ([DateTime]::Now).AddMinutes(10)
+    $s = New-AzAutomationSchedule -ResourceGroupName $rg `
+                                       -AutomationAccountName $aa `
+                                       -Name $name `
+                                       -Description test-OneTime `
+                                       -OneTime `
+                                       -StartTime $startTime `
+                                       -ForUpdate
+
+$query1Scope = @(
+        "/subscriptions/cd45f23b-b832-4fa4-a434-1bf7e6f14a5a/resourceGroups/mms-wcus"
```

Comment:
@jemex as mentioned in the previous PR, we should avoid using hard-coded resource ids where we can -- if it's determined that there is specific setup needed for the test, a comment should be left before the test to outline what the user needs to do in order to properly setup their environment to re-record this test

Reply:
@cormacpayne  I  have added a pre-req steps for re-recording all the tests on the top of the test file